### DISCUSSION TITLE
开发自己的ESLint插件

### DISCUSSION THREAD

**Discussion by giscus** [pinky-pig/arvin#8] [General]
# 开发自己的ESLint插件

ESLint 规则

https://mmeme.me/blog/post/23-%E5%BC%80%E5%8F%91%E8%87%AA%E5%B7%B1%E7%9A%84eslint%E6%8F%92%E4%BB%B6

**Comment by 6owen** [comment]
hei~~~

**Comment by reaiwaris** [comment]
可以可以

**Comment by xiaohanwu520** [comment]
大佬太强了，看了你的很多作品都好有设计感！

**Reply by 6owen** [reply]
哈哈谢谢你😊
